Src/FluentAssertions/Equivalency/Ordering/PathBasedOrderingRule.cs

```diff
@@ -16,6 +16,8 @@ public PathBasedOrderingRule(string path)
         this.path = path;
     }
 
+    public bool Invert { get; set; }
```

Comment:
`Strict` ?

Reply:
`Invert` matches how it's named in `PredicateBasedOrderingRule`.
https://github.com/fluentassertions/fluentassertions/blob/81d3fbf7154f8d9cf80ca0fa72759f863132cf94/Src/FluentAssertions/Equivalency/Ordering/PredicateBasedOrderingRule.cs#L17